todo/todo.go

```diff
@@ -60,10 +51,11 @@ func Read(task_id int) (string, error) {
 }
 
 func ShowAll() error {
-	db := dbConn(user, password, dbname)
+	// db := dbConn(user, password, dbname)
+	db := db.InitDB()
 	defer db.Close()
 
-	statement, err := db.Prepare("SELECT task FROM todo_db;")
+	statement, err := db.Prepare("SELECT task FROM todo_table;")
 	if err != nil {
 		fmt.Println("Enountered error: ", err)
```

Comment:
Why do we need to print this everytime when we are returning the error. we can handle this error on a higher level. Or print the error and not return it. No need to do both of them

Reply:
Line 47 `fmt.Println("", i, " ", task)` Why are we printing `""`?

Reply:
1. Error printing has been removed now.
2. Since this is a CLI app so we are printing these records to console.

Reply:
Why an empty char?

Reply:
Not there anymore. Updated branch here at https://github.com/mohtamohit/go-todo-list/tree/web-cli